agent/async/mocks/async_mocks.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2015-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Looks like this is set in mockgen.sh
https://github.com/aws/amazon-ecs-agent/blob/a902b57c248cc3b8398d55a0608b191e0187b7f2/scripts/generate/mockgen.sh#L29

I can update mockgen.sh if you want - will mean I'm updating the older mocks as well to back date them.